src/lib/components/Link/Link.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { AnchorHTMLAttributes } from "react";
+
+export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {}
+
+const StyledLink = styled.a<LinkProps>`
+	font-size: 14px;
+	font-weight: 700;
+	color: ${({ theme }) => theme.colors.primary.main};
+
+	cursor: default;
+	border-radius: 4px;
+	padding: 4px 4px;
+
+	:is(&:hover, &:focus, &:active) {
```

Comment:
I think it's better to have the following behavior:
1. On hover change only color. 
2. On active, focus change background, don't change the color.